src/components/bs5/globalAlert/globalAlert.function.js

```
@@ -0,0 +1,58 @@
+export function initGlobalAlerts() {
+  const globalAlerts = document.querySelectorAll(".global-alert");
+
+  globalAlerts.forEach((alert) => {
+    const variant = alert.getAttribute("data-variant") || "default";
+    const dismissedAlert = getLocalStorageWithExpiry(
+      `dismissedAlert-${variant}`,
+    );
+
+    if (dismissedAlert) {
+      alert.classList.add("d-none");
+    }
+
+    alert.classList.add("alert");
+    alert.querySelector(".qld-global-alert-main").classList.add("container");
+
+    alert.addEventListener("btn-closed", () => {
+      const expiry = parseInt(alert.getAttribute("data-expiry"), 10);
+      if (expiry && expiry > 0) {
+        setLocalStorageWithExpiry(`dismissedAlert-${variant}`, true, expiry);
```

Similar to above - the key used should be something unique like an ID.  

With fallback to random identifier if the frontend does not provide a unique ID to the component.

---

Hard to keep on an consistant random id, so as we discussed will fallback to dismiss immediately without any expiry